zapgrpc/zapgrpc_test.go

```diff
@@ -33,44 +34,186 @@ import (
 func TestLoggerInfoExpected(t *testing.T) {
 	checkMessages(t, zapcore.DebugLevel, nil, zapcore.InfoLevel, []string{
 		"hello",
-		"world",
+		"hello world",
+		"",
 		"foo",
+		"foo bar",
+		"hello",
+		"hello world",
+		"",
+		"foo",
+		"foo bar",
 	}, func(logger *Logger) {
+		logger.Info("hello")
+		logger.Infof("%s world", "hello")
+		logger.Infoln()
+		logger.Infoln("foo")
+		logger.Infoln("foo", "bar")
```

Comment:
for additional validation, we should add a case with a mix of strings + other types too, e.g.,
```go
log.Infoln("s1", "s2", 1, 2, 3, "s3", 4, "s5", 6)
```